test/mockupdb/test_id_ordering.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from test import PyMongoTestCase
+
+import pytest
+
+from pymongo import InsertOne
+
+try:
+    from mockupdb import MockupDB, OpMsg, go, going
+
+    _HAVE_MOCKUPDB = True
+except ImportError:
+    _HAVE_MOCKUPDB = False
+
+
+from bson.objectid import ObjectId
+
+pytestmark = pytest.mark.mockupdb
+
+
+class TestIdOrdering(PyMongoTestCase):
```

Comment:
Can you add a link to the crud spec that describes this test?

Reply:
Once the spec is merged, yes.

Reply:
Was the spec merged?

Reply:
Yes. added!